src/web/client/common/constants.ts

```diff
@@ -5,8 +5,49 @@
 
 export const PORTAL_LANGUAGES = 'adx_portallanguages';
 export const WEBSITE_LANGUAGES = 'adx_websitelanguages';
+export const WEBSITEID_LANGUAGE = 'adx_websites';
 export const WEBPAGES = 'adx_webpages';
+export const WEBSITES = 'adx_websites';
 export const PORTAL_LANGUAGES_URL_KEY = 'requestUrlForportalLanguage';
 export const WEBSITE_LANGUAGES_URL_KEY = 'requestUrlForWebsitelanguage';
+export const SINGLE_ENTITY_FETCH_URL_KEY = 'fetchUrlSingleEntity';
+export const SINGLE_ENTITY_SAVE_URL_KEY = 'saveUrlSingleEntity';
+export const FETCH_URL_ENTITY_ROOT = 'fetchUrlForEntityRoot';
 export const WEBPAGEID_URL_KEY = 'requestUrlForWebsiteId';
 export const PORTAL_LANGUAGE_DEFAULT = '1033';
+export const PORTALSFOLDERNAME = 'StarterPortal';
+export const PORTALSURISCHEME = 'portals';
+export const PORTALSWORKSPACENAME = 'Power Portals';
+export const WEBPAGES_FILENAME = 'web-pages';
+export const WEBTEMPLATES = 'web-templates';
+export const WEBFILES = 'webfiles';
+export const CONTENTPAGES = 'content-pages';
+export const SCHEMAFILENAME = "PortalSchema.json";
```

Comment:
You can refer this for organizing the constant strings:
https://msazure.visualstudio.com/DefaultCollection/OneAgile/_git/power-platform-ux?path=/packages/powerportals-common/src/constants/PortalConstants.ts